### PR TITLE
Scala add

### PR DESCRIPTION
**Pull request by RichardSmedley**
(leaving version equivalence on 2.x until SDK 3.0 GA)

### CODE REVIEW

File: src/partials/header-content.hbs [hunk 1]
```diff
@@ -38,6 +38,7 @@
                   <li><a href="{{siteRootPath}}/nodejs-sdk/2.6/start-using-sdk.html">Node.js</a></li>
                   <li><a href="{{siteRootPath}}/php-sdk/2.6/start-using-sdk.html">PHP</a></li>
                   <li><a href="{{siteRootPath}}/python-sdk/2.5/start-using-sdk.html">Python</a></li>
+                  <li><a href="{{siteRootPath}}/python-sdk/1.0alpha/start-using-sdk.html">Scala</a></li>
```
Comment (jamesnocentini) [inline src/partials/header-content.hbs]: I think `python-sdk` is a typo?

**Review by jamesnocentini — CHANGES_REQUESTED** [review]
@RichardSmedley Looks good! 👍
I've posted a comment regarding a possible typo.

File: src/partials/header-content.hbs (hunk 1 above)
Comment (RichardSmedley) [reply src/partials/header-content.hbs]: Thanks James, have pushed fix

**Review by jamesnocentini — APPROVED** [review]

File: src/partials/header-content.hbs [hunk 2]
```diff
@@ -38,6 +38,7 @@
                   <li><a href="{{siteRootPath}}/nodejs-sdk/2.6/start-using-sdk.html">Node.js</a></li>
                   <li><a href="{{siteRootPath}}/php-sdk/2.6/start-using-sdk.html">PHP</a></li>
                   <li><a href="{{siteRootPath}}/python-sdk/2.5/start-using-sdk.html">Python</a></li>
+                  <li><a href="{{siteRootPath}}/scala-sdk/1.0alpha/start-using-sdk.html">Scala</a></li>
```
Comment (amarantha-k) [inline src/partials/header-content.hbs:41]: Is it possible to avoid using qualifiers in the URL? This would help SEO in the long run.

**Pull request merged**